docs/data/base/components/table-pagination/TableCustomized.js

```diff
@@ -189,24 +206,43 @@ const CustomTablePagination = styled(TablePagination)(
   }
 
   & .${classes.actions} {
-    padding: 2px;
-    border: 1px solid ${theme.palette.mode === 'dark' ? grey[800] : grey[200]};
-    border-radius: 50px;
+    display: flex;
+    gap: 6px;
+    border: transparent;
     text-align: center;
   }
 
   & .${classes.actions} > button {
-    margin: 0 8px;
+    display: flex;
+    align-items: center;
+    padding: 0;
     border: transparent;
-    border-radius: 2px;
+    border-radius: 99px;
```

Comment:
To avoid arbitrary values:
```suggestion
    border-radius: 50%;
```

\+ x times below